pandas/_testing.py

```diff
@@ -1142,7 +1150,11 @@ def assert_series_equal(
             raise AssertionError("check_exact may only be used with numeric Series")
 
         assert_numpy_array_equal(
-            left._values, right._values, check_dtype=check_dtype, obj=str(obj)
+            left._values,
+            right._values,
+            check_dtype=check_dtype,
+            obj=str(obj),
+            index=left.index._internal_get_values(),
```

Comment:
we are really trying to remove this, just use ``np.asarray(left.index)``

Reply:
done